src/bootstrap.js

```diff
@@ -562,7 +562,7 @@ module.exports = (ceConfig = {}, pnConfig = {}) => {
      * @memberof ChatEngine
      * @see {@link Chat}
      */
-    ChatEngine.Chat = function (...args) {
+    ChatEngine.Chat = (...args) => {
```

Comment:
Is this required for typescript? 

I remember doing this explicitly, I believe in order to do ```new ChatEngine.Chat()``` you need this to be a function. But your test pass, so it's very mysterious.

Reply:
That change was request for lint from the gulp

Reply:
The error I get is "unexpected **unnamed** function". We can get around the error by naming the function something like "createChat". This will solve the error without changing execution context.

Reply:
ok, let me get back this change